src/components/InternalLinkButton/InternalLinkButton.tsx

```diff
@@ -1,19 +1,21 @@
 import Link, { LinkProps } from 'next/link';
 import { Button, ButtonProps } from '@aws-amplify/ui-react';
 
-interface InternalLinkButtonProps extends LinkProps {
+interface InternalLinkButtonProps extends Omit<LinkProps, 'onClick'> {
```

Comment:
I'm omitting this here because the onClick type from next/link isn't the same as the onClick type from Amplify UI Button, and we have to pass the onClick to Button since we're using legacyBehavior on next/link 🙃